config/galaxy.ini.sample

```diff
@@ -1031,6 +1031,11 @@ use_interactive = True
 # if running many handlers.
 #cache_user_job_count = False
 
+#Extra directories to be mounted inside Docker containers. Note that ALL docker containers
+#will mount these volumes
+#extra_docker_volumes = /opt/Genomics:ro,/cluster_share/Galaxy/database/results:rw
```

Comment:
Karthik, Just looking at this. Is the assumption here that when somebody uses this for their standalone install, they can add their /cluster_share/<username> folder to this variable and this will be ignored while pushing their changes back to Galaxy?